nncf/quantization/algorithms/min_max/backend.py

```diff
@@ -83,13 +83,13 @@ def model_transformer(model: TModel) -> ModelTransformer:
 
     @staticmethod
     @abstractmethod
-    def target_point(target_type: TargetType, target_node_name: str, edge_name: str = None) -> TargetPoint:
+    def target_point(target_type: TargetType, target_node_name: str, input_port_id: int = None) -> TargetPoint:
```

Comment:
```suggestion
    def target_point(target_type: TargetType, target_node_name: str, port_id: int = None) -> TargetPoint:
```


I suggest using a more common parameter, because for operations with multiple outputs (`TargetType.POST_LAYER_OPERATION`) `port_id` must also be specified in `TargetPoint`.

Reply:
Actually, we always use 0-index of port_id for TargetType.POST_LAYER_OPERATION.

Reply:
But, I agree that we can align it

Reply:
Look at TF part:
https://github.com/openvinotoolkit/nncf/blob/832c84bb9f2a07a17e1846da14e1bacc54d182a2/nncf/experimental/tensorflow/quantization/algorithm.py#L321-L322

For `OPERATOR_POST_HOOK ` `port_id` is set to 0, but port is used to create `TargetPoint` for both types. You can determine `input_` or `output_` port id by `TargetType`.

Reply:
Done. Also added port_id for Weight type. 
Now port_id is mandatory parameter for creation TargetPoint